src/Components/AddComment.jsx

```diff
@@ -0,0 +1,58 @@
+import {useState} from "react"
+import { PostComment } from "../Api";
+
+function AddComment({review_id, setCommentsList}){
+    
+
+    const [userName, setUserName] = useState("");
+    const [body, setBody] = useState("");
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+       
+        const newComment = {username: userName, body: body}
+
+        PostComment(review_id, newComment).then((newCommentAPI) => {
+            setCommentsList((currComments) => {
+            return [newCommentAPI, ...currComments];
+            });
+
+        });
+
+        setUserName("")
+        setBody("")
+    };
+    
+
+    return (
+    <>
+    <main id="PostCommentContainer">
+
+    <form id="PostCommentForm" onSubmit={handleSubmit}>
+        <h3 id="PostCommentTitle"> Add a comment ⬇️ </h3>
+        <label htmlFor ="username"> Username: </label>
+        <input
+            id="username"
```

Comment:
I'd suggest using useContext to emulate a (even a hardcoded) logged-in user. I'm getting odd behaviours when puting in a username that doesn't exist, but then it occured to me that it would be realy unlikely, in a real-world example, for a user to be able to freely type in any old uername into a comment form! 

Alternatively you could hardcode the user for the comment submission for now until you have user login functionality